app/views/copy_invoices/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<h2><%= "Invoice Payments"%></h2>
+<div>   
+<table>
+    
+     <tr align="left" class="<%= cycle("odd")%>" >
+     <th><strong>Project:</strong></th>
+     <td><%=Project.find(params[:project_id]).name %></td>
+     </tr>
+    </table> 
+</div>
+<div class="autoscroll">
+  <table border="1" style="text-align:center" class="list invoice payments">
+    <thead>
+      <tr>
+        <th> ID</th>
+        <th>Amount</th>
+        <th>Payment Date</th>
+        <th>Invoice Number</th>
+        <th>Description</th>
+        <th>Author</th>
+        <th>Created At</th>
+        <th>Updated At</th>
+        <th>Copy</th>
+
+      </tr>
+    </thead>
+    <tbody>
+      <%  @invoice_payments.each do |invoice_payment| %>
+        <tr class="<%=cycle("odd", "even")%>">
+          <td><%=invoice_payment.id %></td>
```

Comment:
We need to show invoice id here, not invoice_payment id....same for all other fields too